lib/active_record_replica/version.rb

```diff
@@ -1,3 +1,3 @@
 module ActiveRecordReplica
-  VERSION = '2.0.2'
+  VERSION = '2.1.0'
```

Comment:
Perhaps this should be v3 because this a non-backwards compatible change?
```suggestion
  VERSION = '3.0.0'
```